core/string_api.php

```diff
@@ -644,9 +644,10 @@ function string_get_bug_page( $p_action ) {
  * @param integer $p_bug_id	     A bug identifier.
  * @param boolean $p_detail_info Whether to include more detailed information (e.g. title attribute / project) in the returned string.
  * @param boolean $p_fqdn        Whether to return an absolute or relative link.
+ * @param boolean $p_long_link 	 When set to false, the returned link is displayed as bug ID. When set to true, the return string is displayed as ID: summary.
```

Comment:
Thank you for the correction. I had corrected this to '<ID>:<space><ShortenedSummary>'